packages/web/src/components/RedwoodApolloProvider.tsx

```diff
@@ -20,12 +23,46 @@ const ApolloProviderWithFetchConfig: React.FunctionComponent<{
   config?: Omit<ApolloClientOptions<InMemoryCache>, 'cache'>
 }> = ({ config = {}, children }) => {
   const { uri, headers } = useFetchConfig()
+  const { getToken, type: authProviderType, isAuthenticated } = useAuth()
+
+  const withToken = setContext(async () => {
+    if (isAuthenticated && getToken) {
```

Comment:
Is there a case when `getToken` would be falsy?

Reply:
This was causing the failure in the e2e test. Its falsy when you're not logged in - wasn't sure if there's any other case where its falsy apart from the first render of the auth provider.

Reply:
getToken is falsey? But surely that shouldn't be the case since the function is provided via the provider? Maybe one of the "fake implementations" doesn't follow the providers spec.